core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -131,9 +131,9 @@
   public static final String MESSAGE_ACCEPT_ENCODING = "grpc-accept-encoding";
 
   /**
-   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 100 MiB.
+   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 4 MiB.
    */
-  public static final int DEFAULT_MAX_MESSAGE_SIZE = 100 * 1024 * 1024;
+  public static final int DEFAULT_MAX_MESSAGE_SIZE = 4 * 1024 * 1024;
```

Comment:
You might also consider making this a literal.  When we print out the error that the max message size is _______, its nice to be able to copy paste that number and search for it.